### ISSUE TITLE
Missing headings for additional accessibility information

### ISSUE THREAD

**Issue by mattgarrish** [w3c/publ-a11y#502]
As the title says, I don't see any instructions in 4.10 for outputting headings for the section or its subsections.

As it is, all that gets displayed are the comma-joined strings.

Should the heading be "Additional accessibility information" and "Adaptation" and "Clarity" are subheadings, or are the subsections missing sentence fragments that would make the strings understandable?

Applies to both techniques documents.
